### ISSUE TITLE
Login page is not responsive, has no footer and no placeholder value for input

### ISSUE THREAD

**Issue by Ayush215mb** [ChandelAnish/hireHUB-website#41]
Your website's login page is not responsive for small screens and the UI is also not good enough, there is no footer and even the placeholder values are missing in the input section.
I can do all of this in 2-3 hours and create a PR with a better Login page, so please assign me this issue as soon as possible.

**Comment by ChandelAnish** [comment]
make it responsive, no need to add footer

**Comment by Ayush215mb** [comment]
shall i make a better UI as well or just make the existing UI responsive?

**Comment by Ayush215mb** [comment]
![image](https://github.com/user-attachments/assets/bca913af-b44b-4e21-b15d-7e48a07bec02)
![image](https://github.com/user-attachments/assets/c4e16388-41e4-4a53-a28a-4f785bb3f12b)
I have done these changes, do you want me to make any other changes or shall I create a PR request??

**Comment by ChandelAnish** [comment]
you need to first sync to latest changes which are committed on the dev-gssoc branch and then PR on dev-gssoc branch

**Comment by Ayush215mb** [comment]
yeah I did that right now and have added my own changes as well in the dev-gssoc branch now I'm creating the PR in dev-gssoc branch

**Comment by ChandelAnish** [comment]
share the screenshots

**Comment by Ayush215mb** [comment]
![image](https://github.com/user-attachments/assets/d14337dd-4559-4ed3-a33f-ebe911581549)
![image](https://github.com/user-attachments/assets/3aa69512-2b6b-4d55-b0df-48c355c83e3b)

**Comment by Ayush215mb** [comment]
@ChandelAnish I have created the PR in dev-gssoc branch as you said.
please review it asap.

**Comment by Ayush215mb** [comment]
@ChandelAnish Please review it.

**Comment by ChandelAnish** [comment]
The eye icon is overflowing fix it and I'll merge
